monodbassignment/posts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id":"String datatype", 
+  "author_id": "String datatype(id of author)",
+   "timestamp":"timestamp datatype",
+  "content":{
```

Comment:
this sub document is unnecessary, you would never have to fetch these details separately.